h2o-core/src/main/java/water/util/MRUtils.java

```diff
@@ -128,6 +130,57 @@ public void map(Chunk[] cs, NewChunk[] ncs) {
     @Override public void reduce( ClassDist that ) { ArrayUtils.add(_ys,that._ys); }
   }
 
+  /**
+   * Compute the class distribution for qusibinomial distribution from a class label vector
+   * (not counting missing values)
+   */
+  public static class ClassDistQuasibinomial extends MRTask<ClassDistQuasibinomial> {
+    final int _nclass;
+    protected double[] _ys;
+    protected Double[] _domain;
+    
+    public ClassDistQuasibinomial(String[] domain) { 
+      _nclass = 2;
+      _domain = new Double[]{Double.valueOf(domain[0]), Double.valueOf(domain[1])};
```

Comment:
seems like the code only needs the first element of the array

Reply:
True. Fixed.